vpr/src/server/telegrambuffer.h

```diff
@@ -44,14 +49,14 @@ class TelegramBuffer
      * 
      * @param data The byte array whose contents will be appended to internal byte buffer.
      */
-    void append(const ByteArray&);
+    void append(const ByteArray& data);
 
     /**
      * @brief Extracts well-formed telegram frames from the internal byte buffer.
      * 
      * @param frames A reference to a vector where the extracted telegram frames will be stored.
      */
-    void take_telegram_frames(std::vector<TelegramFramePtr>&);
+    void take_telegram_frames(std::vector<TelegramFramePtr>& frames);
 
     /**
      * @brief Extracts well-formed telegram frames from the internal byte buffer.
```

Comment:
Why are both versions of take_telegram_frames needed?

Reply:
take_telegram_frames(std::vector<TelegramFramePtr>& frames) allows to optimize code by
avoids frequent allocation std::vector<comm::TelegramFramePtr>. It uses inside the comm std::thread event loop.
take_telegram_frames with no argument is used in tests, because it allow write test scenario code with one line less.
Anyway i am going to remove slow variant of function call and modify test cases.

Reply:
solved 138f8b729ae5c2895a9f1d09abf4a6db109c9da3